.github/workflows/tests.yml

```diff
@@ -236,6 +230,12 @@ jobs:
         if: "${{ matrix.node-version == 'v14.x' }}"
         run: npm uninstall @typescript-eslint/eslint-plugin eslint-plugin-prettier tap && npm install tap@16 ts-node && npx npm@7 pkg delete tap
 
+      - name: Setup self-direct dependency
+        run: npm link
+
+      - name: Link self-direct dependency
+        run: npm link @google-cloud/cloud-sql-connector
+
```

Comment:
moved this block since it was interfering with the `npm uninstall` from the `Setup v14.x-compatible tap` step above